Sources/UINavigationController.swift

```diff
@@ -33,14 +33,17 @@ open class UINavigationController: UIViewController {
 
     private func updateUIFromViewControllerStack(animated: Bool) {
         guard _view != nil else { return }
+        if presentedViewController === viewControllers.last { return }
 
         transitionView.subviews.forEach { $0.removeFromSuperview() }
         presentedViewController = viewControllers.last
 
         if let viewOnTopOfStack = viewControllers.last?.view {
             // TODO: Animate here
             viewOnTopOfStack.frame = transitionView.bounds
+            presentedViewController?.viewWillAppear(animated)
             transitionView.addSubview(viewOnTopOfStack)
+            presentedViewController?.viewDidAppear(animated)
```

Comment:
also these changes aren't related to this PRs intention.

Reply:
sorry the PR base was wrong. based this on the previous PR because I plan to merge it first